src/app/games/diaballik/tests/DiaballikFilteredMoveGenerator.spec.ts

```diff
@@ -1,27 +1,38 @@
-import { DiaballikBallPass, DiaballikMove, DiaballikSubMove, DiaballikTranslation } from '../DiaballikMove';
+import { DiaballikBallPass, DiaballikMove, DiaballikSubMove, DiaballikTranslation, isTranslation } from '../DiaballikMove';
 import { DiaballikFilteredMoveGenerator } from '../DiaballikFilteredMoveGenerator';
 import { DiaballikNode } from '../DiaballikRules';
 import { DiaballikState } from '../DiaballikState';
 
-describe('DiaballikFilteredMoveGenerator', () => {
+function expectNoBackAndForth(move: DiaballikMove): void {
+    const subMoves: DiaballikSubMove[] = move.getSubMoves();
+    const translations: DiaballikSubMove[] = subMoves.filter(isTranslation);
+    const startIsEnd: boolean = translations[0].getStart() === translations[1].getEnd();
+    const endIsStart: boolean = translations[0].getEnd() === translations[1].getStart();
```

Comment:
"start is end, end is start, life is death, poems are letters."
Sir Jean Jaja de la Jã, 2 janvier 1492.